### DISCUSSION TITLE
What is the recommended way to use the latest (Github) version of the Python package?

### DISCUSSION THREAD

**Discussion by vibl** [taskforcesh/bullmq#1908] [Q&A] [answered]
As I signaled on https://github.com/taskforcesh/bullmq/issues/1903, the Python build script (setup.py) is broken (and consequently, so is `pip install -e 'git+https://github.com/taskforcesh/bullmq#egg=bullmq&subdirectory=python'`).

I was told that "The setup.py is only intended for doing new releases of the package, i.e., used by the maintainers to update the package in pypi.", and the issue was closed.

So if the build script is not the recommended way to use the latest (Github) version of the Python package, what is?

I can't use the latest Pypi release (0.3.0) because it cannot use Redis with SSL. Only the newest code on Github can.

**Comment by manast** [comment, marked as answer]
Here version 0.4.0: https://pypi.org/project/bullmq/0.4.0/
